documentation/SA1649.md

```diff
@@ -49,6 +49,10 @@ To fix a violation of this rule, rename the file to  match the name of the first
 
 ## How to suppress violations
 
+```csharp
+[SuppressMessage("StyleCop.CSharp.DocumentationRules", "SA1649:FileNameMustMatchTypeName", Justification = "Reviewed.")]
```

Comment:
❓ Does this work? If so, can you add a test in [SA1649UnitTests.cs](https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/5ba5c03f1ddb302c38771a6b5615ec5142ef8c8a/StyleCop.Analyzers/StyleCop.Analyzers.Test/DocumentationRules/SA1649UnitTests.cs) showing that this syntax will prevent the issue from being reported?